<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1039</width>
    <height>708</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string/>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <widget class="QSplitter" name="splitter">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <widget class="ImageListWidget" name="imageListWidget" native="true"/>
      <widget class="PageListWidget" name="pageListWidget" native="true"/>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1039</width>
     <height>22</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuProject">
    <property name="title">
     <string>File</string>
    </property>
    <widget class="QMenu" name="menuExport">
     <property name="title">
      <string>Export</string>
     </property>
     <addaction name="actionPs"/>
     <addaction name="actionPdf"/>
    </widget>
    <addaction name="actionNew"/>
    <addaction name="actionOpen"/>
    <addaction name="actionSave"/>
    <addaction name="actionSave_as"/>
    <addaction name="actionClose"/>
    <addaction name="separator"/>
    <addaction name="menuExport"/>
    <addaction name="separator"/>
    <addaction name="actionExit"/>
   </widget>
   <widget class="QMenu" name="menuOptions">
    <property name="title">
     <string>Options</string>
    </property>
    <widget class="QMenu" name="menuEditor">
     <property name="title">
      <string>Editor</string>
     </property>
     <addaction name="actionAdd"/>
     <addaction name="actionClear"/>
     <addaction name="actionLayout"/>
     <addaction name="actionShow_date_first_of_month"/>
     <addaction name="actionResort_by_date"/>
    </widget>
    <addaction name="menuEditor"/>
    <addaction name="actionBook"/>
   </widget>
   <widget class="QMenu" name="menuAbout">
    <property name="title">
     <string>About</string>
    </property>
   </widget>
   <addaction name="menuProject"/>
   <addaction name="menuOptions"/>
   <addaction name="menuAbout"/>
  </widget>
  <action name="actionSave">
   <property name="text">
    <string>Save</string>
   </property>
   <property name="toolTip">
    <string>Save project</string>
   </property>
  </action>
  <action name="actionOpen">
   <property name="text">
    <string>Open</string>
   </property>
   <property name="toolTip">
    <string>Open project</string>
   </property>
  </action>
  <action name="actionAdd">
   <property name="text">
    <string>Add images</string>
   </property>
   <property name="toolTip">
    <string>Add images form directory</string>
   </property>
  </action>
  <action name="actionClear">
   <property name="text">
    <string>Clear images</string>
   </property>
   <property name="toolTip">
    <string>Clear images list</string>
   </property>
  </action>
  <action name="actionLayout">
   <property name="text">
    <string>Layout</string>
   </property>
   <property name="toolTip">
    <string>Layout settings</string>
   </property>
  </action>
  <action name="actionSave_as">
   <property name="text">
    <string>Save as</string>
   </property>
  </action>
  <action name="actionClose">
   <property name="text">
    <string>Close</string>
   </property>
  </action>
  <action name="actionExit">
   <property name="text">
    <string>Exit</string>
   </property>
  </action>
  <action name="actionNew">
   <property name="text">
    <string>New</string>
   </property>
  </action>
  <action name="actionPs">
   <property name="text">
    <string>Ps</string>
   </property>
  </action>
  <action name="actionPdf">
   <property name="text">
    <string>Pdf</string>
   </property>
  </action>
  <action name="actionShow_date_first_of_month">
   <property name="text">
    <string>Show date first of month</string>
   </property>
  </action>
  <action name="actionBook">
   <property name="text">
    <string>Book</string>
   </property>
  </action>
  <action name="actionImages">
   <property name="text">
    <string>Images</string>
   </property>
  </action>
  <action name="actionResort_by_date">
   <property name="text">
    <string>Resort by date</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>ImageListWidget</class>
   <extends>QWidget</extends>
   <header>imagelistwidget.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>PageListWidget</class>
   <extends>QWidget</extends>
   <header>pagelistwidget.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
